### ISSUE TITLE
Revise `jsx-enforce-attribute` rule name

### ISSUE THREAD

**Issue by ZebulanStanphill** [DmitriyNikolenko/eslint-plugin-jsx-helpers#2]
The rules in [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y) include neither `jsx` nor `a11y` in their rule names, because their `jsx-a11y` namespace would make that redundant. Similarly, [`eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react) does not have its rule names start with `react`.

Additionally, there seems to be a naming convention across most ESLint rules where a rule intended to require something to be present is called `require-...`. Another convention among rules similar to this plugin's `jsx-enforce-attribute` is using plural case, e.g. `react/forbid-component-props` or `react/jsx-no-duplicate-props`.

For these reasons, I propose renaming the `jsx-helpers/jsx-enforce-attribute` rule to simply `jsx-helpers/require-attributes` for consistency with ESLint rules in other plugins.

**Comment by DmitriyNikolenko** [comment]
Sounds logically enough, I may do it as soon as found time.
